flag.go

```diff
@@ -596,6 +596,8 @@ func UnquoteUsage(flag *Flag) (name string, usage string) {
 		name = "int"
 	case "uint64":
 		name = "uint"
+	case "stringArray":
```

Comment:
Unfortunately, this will have an adverse effect on github.com/spf13/viper, which relies on those values. 
See there: https://github.com/spf13/viper/blob/b89e554a96abde447ad13a26dcc59fd00375e555/viper.go#L1262

Reply:
That code looks incorrect to me: viper tries to read in a `StringArray` as multiple comma-separated values, but [the documentation](https://github.com/spf13/pflag/blob/v1.0.5/string_array.go#L122) explicitly states that `StringArray` is not treated as multiple comman-separated values:

> The value of each argument will not try to be separated by comma.